pkg/cli/admin/toppvc/persistentvolumeclaims.go

```diff
@@ -255,6 +256,15 @@ func getWithBearer(ctx context.Context, getRoute RouteGetter, namespace, name st
 		TLSClientConfig: &tls.Config{InsecureSkipVerify: InsecureTLS},
 	}
 
+	proxyEnv := os.Getenv("https_proxy")
```

Comment:
I believe this should be
```suggestion
	proxyEnv := os.Getenv("HTTPS_PROXY")
```
Getenv is case sensitive.

Reply:
Good question, generally Lowercase `https_proxy`: Used by tools like `curl`, `wget`, and many Unix utilities. In CI We also use lowercase export the proxy and use oc access proxy clusters.
Uppercase `HTTPS_PROXY`: Used by tools like `npm`, `yarn`, and some Java applications.  It is up to how we doc this.

Reply:
Example commit where I used this option before:
https://github.com/IBM/ibmcloud-volume-interface/pull/38/commits/53fdceea7a2d07bb98611a594c97f835c8a7255e#diff-b494db4ae8b0cc8a58ac4ca3b9d3bd5e69159561c55dc6e361cf828b70b47616R40

Reply:
@dobsonj Cool, that will be much better! Fixed, thank you so much for the good tips!